userdocs/creating-a-codelist-from-a-csv-file.md

```diff
@@ -18,23 +18,11 @@ If you are a member of an organisation, you can also choose an owner for the cod
 
         create-codelist-button.png
 
-### Requirements for uploading codelists to OpenCodelists
+### Requirements for uploading codelists to OpenCodelists via the "Create a codelist" form
 
 * Store the final codelist in CSV format.
-* Store codes in exactly one column.
+* Store codes in the first column.
 * Check whether or not your codelist has a header row: you will need to specify this on upload.
-* There is currently a soft requirement that the first column must contain codes in the chosen coding system. These should preferably be named according to the CSV column names provided in the table above. (We plan to eventually remove this requirement.) The second column is typically a text description of the code.
-
-### More about codelist columns
-
-The codelist page allow you to upload two columns:
-
-* a code
-* a text description of the code
-
-However, some codelists may require a 'classification' or 'type' column, which classifies the codes into subcategories. For example, when using a codelist for venous thromboembolism, you may wish to classify these codes into deep vein thromboses and pulmonary embolisms. By using subcategories, you can keep all the codes in a single codelist, rather than uploading separate lists for each clinical subcategory. The OpenSAFELY documentation has [guidance for using category columns with OpenSAFELY ehrQL dataset definition](https://docs.opensafely.org/ehrql/how-to/examples/#using-codelists-with-category-columns).
-
-**Uploading more than two columns is currently only possible for the OpenSAFELY core team. If your study requires this feature, [please get in touch](https://www.opensafely.org/contact/).**
```

Comment:
thought: I'm not entirely sure about removing this section. Are some users working this way intentionally, using the `/add` upload form instead of the main "Create a codelist" form?

We could move it into the discussion on OPCS-4 codelists, and retitle the section "using the alternative/legacy upload form"?